packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
Can we put this code in a library that we then link to? We're not including all of the engine code, for example. Can this code belong there? That's what we've done on the iOS side.